### DISCUSSION TITLE
第4节课笔记(11班）

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#240] [笔记]
加油加油~~~ 记得打完卡要在群里打分表登记哟。

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/676540716

**Comment by zhanghui-china** [comment]
https://zhuanlan.zhihu.com/p/677117479

**Comment by LichKing-W** [comment]
https://zhuanlan.zhihu.com/p/677350038

**Comment by qianlian-mozi** [comment]
https://blog.csdn.net/m0_54824511/article/details/135542819

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135537980

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135513305

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E5%9B%9B%E8%8A%82.md

**Comment by Unicornlyy** [comment]
https://blog.csdn.net/m0_68165821/article/details/135551666

**Comment by Kongsea** [comment]
笔记第四课：https://blog.csdn.net/deeepsea/article/details/135545131

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/note/4/%E7%AC%AC%E5%9B%9B%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/677337813?

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/4%E3%80%81XTuner-%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%8D%95%E5%8D%A1%E4%BD%8E%E6%88%90%E6%9C%AC%E5%BE%AE%E8%B0%83%E5%AE%9E%E6%88%98

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/955a693e-0148-4415-9e2f-e31636dea662
【FlowUs 息流】XTuner 大模型单卡低成本微调

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/4.md

**Comment by Vacili-f** [comment]
https://github.com/Vacili-f/InternLM/blob/main/%E7%AC%94%E8%AE%B0/%E7%AC%AC%E5%9B%9B%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by Lyusijie** [comment]
https://blog.csdn.net/weixin_44986362/article/details/135578938?spm=1001.2014.3001.5501

**Comment by hellokid123** [comment]
https://github.com/hellokid123/internllm-notes/blob/main/notes/%E7%AC%AC4%E8%AF%BE.md

**Comment by Tangent-90C** [comment]
https://github.com/Tangent-90C/public-note/tree/main/%E5%BE%AE%E8%B0%83%E4%B8%AA%E8%87%AA%E5%B7%B1%E7%9A%84%E7%A7%98%E4%B9%A6
